.gitignore

```diff
@@ -1,17 +1,16 @@
-*.pyc
-.DS_Store
-*.swp
+# Be sure to update .dockerignore to match .gitignore
 /venv/
 /static/
 /media/
-/docs/_build/
-/ietf/static/node_modules
-/ietf/static/yarn-error.log
```

Comment:
This entry is not in the new updated ignore files.

Reply:
There's already a generic `*.log` entry that will ignore `/ietf/static/yarn-error.log` too.